### ISSUE TITLE
Please add eslint-plugin-vue-a11y support

### ISSUE THREAD

**Issue by robbymarston** [houndci/hound#1670]
I'm getting this error when trying to use [eslint-plugin-vue-a11y](https://github.com/maranran/eslint-plugin-vue-a11y):

```
Oops! Something went wrong! :(
ESLint: 4.19.1.
ESLint couldn't find the plugin "eslint-plugin-vue-a11y". This can happen for a couple different reasons:
1. If ESLint is installed globally, then make sure eslint-plugin-vue-a11y is also installed globally. A globally-installed ESLint cannot find a locally-installed plugin.
2. If ESLint is installed locally, then it's likely that the plugin isn't installed correctly. Try reinstalling by running the following:
    npm i eslint-plugin-vue-a11y@latest --save-dev
If you still can't figure out the problem, please stop by https://gitter.im/eslint/eslint to chat with the team.
```

If you could add support for the plugin it'd be much appreciated, thank you!

**Comment by gylaz** [comment]
We've added support for this package. Let us know if you still see this issue.
